pkg/controller/studyjob/studyjob_controller.go

```diff
@@ -226,11 +227,6 @@ func (r *ReconcileStudyJobController) Reconcile(request reconcile.Request) (reco
 		if !contains(pendingFinalizers, cleanDataFinalizer) {
 			return reconcile.Result{}, nil
 		}
-		err = deleteStudy(instance)
```

Comment:
I think maybe we can add a field like "Retain" for studyjob to decide if deleteStudy when delete studyjob